store/src/java/com/zimbra/cs/service/mail/AnnounceOrganizerChange.java

```diff
@@ -53,12 +54,11 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
 
         MailSendQueue sendQueue = new MailSendQueue();
         Element response = getResponseElement(zsc);
-        mbox.lock.lock();
-        try {
+        try (final MailboxLock l = mbox.lock(true)) {
```

Comment:
Control flow was changed here. `l` needs to be closed before `sendQueue.send()` is called. You can manage the lifecycle of `l` yourself of your can do something like this.

```java
try {
    try (final MailboxLock l = mbox.lock(true)) {
        l.lock();
        CalendarItem calItem = mbox.getCalendarItemById(octxt, iid,getId());
        sendOrganizerChangeMessage(zsc, octxt, calItem, acct, mbox, sendQueue);
    }
} finally {
    sendQueue.send();
}
```